src/common/utils/token-utils.ts

```diff
@@ -0,0 +1,19 @@
+import { isValidUrl } from '@common/validation/validate-url';
+import { FungibleTokenMetadata } from '@stacks/blockchain-api-client';
+import { convertUnicodeToAscii } from './string-utils';
+
+export const isIconUrl = isValidUrl;
```

Comment:
why do we need to assign the function to a new constant?

Reply:
It was just for semantic, it's not relevant anymore as I changed the logic